### PR TITLE
Backport default.yaml split patches

### PR DESCRIPTION
**Pull request by raukadah**
It backports following patches:

* https://github.com/openstack-k8s-operators/architecture/pull/375 
* https://github.com/openstack-k8s-operators/architecture/pull/383 
* https://github.com/openstack-k8s-operators/architecture/pull/379 
* https://github.com/openstack-k8s-operators/architecture/pull/384 
* https://github.com/openstack-k8s-operators/architecture/pull/373

### CODE REVIEW

**Comment by fultonj** [comment]
This looks fine to me. @cjeanner @eduolivares @eshulman2 please let me know when you think it's safe to merge.

**Comment by cjeanner** [comment]
Once we get proper testing results, we should be able to merge. I've updated the Depends-On on the ci-framework PR that backports the needed change as well.

File: automation/vars/nfv-ovs-dpdk-sriov-hci.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            --timeout=60m
+        values:
+          - name: edpm-deployment-values
+            src_file: values.yaml
+        build_output: deployment-pre-ceph.yaml
+        post_stage_run:
+          - name: Deploy Ceph
+            type: playbook
+            source: "../../playbooks/ceph.yml"
+            inventory: "${HOME}/ci-framework-data/artifacts/zuul_inventory.yml"
```
Comment (eshulman2) [inline automation/vars/nfv-ovs-dpdk-sriov-hci.yaml:54]: please match this file content with https://github.com/openstack-k8s-operators/architecture/blob/main/automation/vars/nfv-ovs-dpdk-sriov-hci.yaml

**Comment by eshulman2** [comment]
lgtm

**Comment by eduolivares** [comment]
lgtm
please merge together with https://github.com/openstack-k8s-operators/ci-framework/pull/2333

**Review by cjeanner — APPROVED** [review]
/lgtm
/approve

I took care of the ci-framework - will force-merge if prow's too slow there.

**Pull request merged**